### PR TITLE
PWM Steering fixed

### PR DESCRIPTION
**Pull request by benjaf**
PWM mode steering channel now uses same timer as throttle, which works a lot better. STEER_COEFICIENT set to 1.0 by default.

### CODE REVIEW

**Comment by EmanuelFeru** [comment]
Hi @benjaf,

I did a quick update (see the commit changes). I hope you agree with them.

File: Src/control.c [hunk 1]
```diff
@@ -127,34 +115,34 @@ void PWM_ISR_CH2_Callback() {
 }
 
 // SysTick executes once each ms
-void PWM_SysTick_Callback() {
+void PWM_SysTick_Callback(void) {
   pwm_timeout++;
   // Stop after 500 ms without PPM signal
   if(pwm_timeout > 500) {
     //pwm_captured_ch1_value = 500;
```
Comment (EmanuelFeru) [inline Src/control.c:122]: Why is this commented-out. Shouldn't we put also ch1 (Steering) to 500 which will results to cmd1 = 0, in case of Timeout?

Comment (benjaf) [reply Src/control.c:122]: Yes, we should. I missed that one!

File: Src/control.c [hunk 2]
```diff
@@ -127,34 +115,34 @@ void PWM_ISR_CH2_Callback() {
...
     pwm_captured_ch2_value = 500;
-    pwm_timeout = 0;
+    pwm_timeout = 500;
```
Comment (benjaf) [inline Src/control.c:124]: This acts as a reset of the PWM timeout, allowing it to retry after the safe values have been set. It is supposed to be 0.

Comment (EmanuelFeru) [reply Src/control.c:124]: Ok.

**Comment by EmanuelFeru** [comment]
Fixed Issue #56.

**Pull request merged**